### PR TITLE
Feature: Pyttsx

### PR DESCRIPTION
**Pull request by Supreme-hub**
# Description
Allows the user to use Pyttsx for TTS.

# Issue Fixes

#1063 

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have formatted and linted my code using python-black and pylint
- [x] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

**Comment by Supreme-hub** [comment]
Hey, I changed the branch name, so it closed the PR.

**Review by CallumIO — CHANGES_REQUESTED** [review]
Please format your files using black

You have changed every file for a simple addition

**Comment by Supreme-hub** [comment]
Hey, black made those changes, not me

**Comment by Supreme-hub** [comment]
I have done the changes you asked

**Comment by Supreme-hub** [comment]
@CallumIO

File: utils/.config.template.toml [hunk 1]
```diff
@@ -31,7 +31,9 @@ background_choice = { optional = true, default = "minecraft", example = "rocket-
 
 
 [settings.tts]
-voice_choice = { optional = false, default = "", options = ["streamlabspolly", "tiktok", "googletranslate", "awspolly", ], example = "tiktok", explanation = "The voice platform used for TTS generation. This can be left blank and you will be prompted to choose at runtime." }
+voice_choice = { optional = false, default = "", options = ["streamlabspolly", "tiktok", "googletranslate", "awspolly", "pyttsx"], example = "tiktok", explanation = "The voice platform used for TTS generation. This can be left blank and you will be prompted to choose at runtime." }
```
Comment (JasonLovesDoggo) [inline utils/.config.template.toml:34]: Add back the trailing comma

**Review by JasonLovesDoggo — CHANGES_REQUESTED** [review]

**Comment by Supreme-hub** [comment]
Added back trailing commas

**Review by JasonLovesDoggo — APPROVED** [review]

**Pull request merged**